project_recalculate/models/project_project.py

```diff
@@ -64,10 +64,9 @@ def project_recalculate(self):
                     project.date):
                 raise Warning(_("Cannot recalculate project because your "
                                 "project don't have date end."))
-            if project.calculation_type != 'none':
-                for task in project.tasks:
-                    task.task_recalculate()
-                vals = project._start_end_dates_prepare()
-                if vals:
-                    project.write(vals)
+            for task in project.tasks:
```

Comment:
This should be in English.

But if you have the Warning, what is the problem then?